src/blocks/types.tsx

```diff
@@ -29,9 +29,13 @@ import CameraIcon from './blockIcons/CameraIcon.png'
 import voiceNoteIcon from './blockIcons/voiceNoteIcon.png'
 import CalligraphyIcon from "./blockIcons/calligraphyIcon.png"
 import localelocatrIcon from "./blockIcons/localelocatrIcon.png";
+import distrivialIcon from "./blockIcons/distrivialIcon.png";
 import vibecheckIcon from "./blockIcons/vibecheckIcon.png";
 
 export type BlockModel = {
+  userAnswers: never[];
```

Comment:
Rather than adding new fields to the model (which would break the data storage for other miniapps), you should add and access data from the model's data property, like so:
```
model.data.userScore = userScore.toString()
model.data.questions = JSON.stringify(questions)
model.data.userAnswers = JSON.stringify(userAnswers)
```
This is a more extensible way to store your data and makes the miniapp model work for everyone!